### PR TITLE
Task6/implementing theme

### PR DESCRIPTION
**Pull request by Silvik95**
image
in this task was implemented 2 themes: dark and light

### CODE REVIEW

File: src/components/Header.jsx [hunk 1]
```diff
@@ -1,8 +1,28 @@
...
+  const dispatch = useDispatch();
+  function switchTheme(themeName) {
+    dispatch(setTheme(themeName));
+  }
+
+  useEffect(() => {
+    const backgroundColorHeader = `var(--background-color-${theme})`;
+    const backgroundColorMain = `var(--background-color-main-${theme})`;
+    document.body.style.setProperty('--background-color', backgroundColorHeader);
+    document.body.style.setProperty('--background-color-main', backgroundColorMain);
```
Comment (virzhinia-petkova-redshelf) [inline src/components/Header.jsx]: are you only switching the background color?

Comment (Silvik95) [reply src/components/Header.jsx]: Yep, all text colors looks good in both themes

Comment (Silvik95) [reply src/components/Header.jsx]: I can switch other colors if it needs

**Review by virzhinia-petkova-redshelf — COMMENTED** [review]
lets remove commented out code and we can merge it

**Review by virzhinia-petkova-redshelf — APPROVED** [review]

**Pull request merged**